src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#smth" class="logo">
+        <img src="./images/logo.png"
+          alt="image1"
+          class="image1"
+        >
```

Comment:
if more then 3 attributes - put each on new line
make sure to fix it here and everywhere else in your code
```suggestion
        <img 
          src="./images/logo.png"
          alt="image1"
          class="image1"
        >
```